### DISCUSSION TITLE
What does these settings do?

### DISCUSSION THREAD

**Discussion by ghost** [Acode-Foundation/Acode#717] [Q&A] [answered]
![Screenshot_20230612-164501_Acode](https://github.com/deadlyjack/Acode/assets/113766442/743e350d-a314-4aa1-8c19-eabefdde5191)
![Screenshot_20230612-164519_Acode](https://github.com/deadlyjack/Acode/assets/113766442/dfbb3e1d-c4cd-4540-82ff-eb1e9708a3b3)
![Screenshot_20230612-164817_Acode](https://github.com/deadlyjack/Acode/assets/113766442/b8245aad-9022-4620-97b3-745821cd1489)
![Screenshot_20230612-164835_Acode](https://github.com/deadlyjack/Acode/assets/113766442/4bd4a880-0023-4d88-a715-4a989f5fd283)

**Comment by ghost** [comment]
@deadlyjack

**Reply by deadlyjack** [reply, marked as answer]
These are ace editor settings. For better understanding Google them.
